todoredux/src/App.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./App.css";
+import { connect } from "react-redux";
+import { getTodo, addTodo } from "./action/addTodo";
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: "",
+      data: [],
+      onclick: false
```

Comment:
what is this use proper variable name

Reply:
-complete form validation,data show in the table, update data show in table.

Reply:
ok sir